default/stringtables/en.txt

```diff
@@ -3918,6 +3918,15 @@ Outpost
 OUTPOSTS_TEXT
 Outposts are unmanned stations, which can be founded in places too hostile for a colony to survive. They have no population, and normally produce no resources. But they create [[encyclopedia SUPPLY_TITLE]] (with the right tech) and provide vision. Colonies can be founded on planets that already have an Outpost. To build an Outpost, you need to put an [[shippart CO_OUTPOST_POD]] on a ship. Outposts are also formed when the population of a colony drops to zero for any reason.
 
+FOCUS_TITLE
+Planetary Focus
+FOCUS_TEXT
+'''Each planet can be focused on a specific area, such as production or research.  Not all areas of focus are always available.  They may be restricted because the species can not focus on that area, the technology is not yet known, or other factors such as the availability of a special.
+
+The focus of a planet may grant it bonuses or abilities, depending on the focus.  Changing a planets focus occurs a small penalty that will diminish over time.
```

Comment:
"abilities" seems like odd word choice... "effects" might be better.

"The focus ... the focus." is awkward.